jenkins_jobs/trigger_nightly_host_os_build/post_build_script.sh

```diff
@@ -1,16 +1,17 @@
 NIGHTLY_DIR_NAME=$(cat NIGHTLY_DIR_NAME)
 BUILD_TIMESTAMP=$(cat BUILD_TIMESTAMP)
 BUILD_DIR_PATH="../builds/${BUILD_TIMESTAMP}"
+NIGHLTY_DIR_RSYNC_URL="${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${UPLOAD_SERVER_NIGHTLY_DIR}"
+
+rsync_upload() {
+    rsync -e "ssh -i ${HOME}/.ssh/upload_server_id_rsa" \
+          --verbose --compress --links --stats --times --chmod=a+rwx,g+wx,o- \
+          $@ $NIGHLTY_DIR_RSYNC_URL
+}
 
 # update nightly build dir and latest nightly build
 ln -s "$BUILD_DIR_PATH" "$NIGHTLY_DIR_NAME"
 ln -s "$NIGHTLY_DIR_NAME" latest
 
-rsync -e "ssh -i $HOME/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
-      --verbose --compress --links --times --chmod=a+rwx,g+rwx,o- \
-      "$NIGHTLY_DIR_NAME" \
-      "${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${UPLOAD_SERVER_NIGHTLY_DIR}/"
-rsync -e "ssh -i $HOME/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa" \
-      --verbose --compress --links --times --chmod=a+rwx,g+rwx,o- \
-      "latest" \
-      "${UPLOAD_SERVER_USER_NAME}@${UPLOAD_SERVER_HOST_NAME}:${UPLOAD_SERVER_NIGHTLY_DIR}/"
+rsync_upload "$NIGHTLY_DIR_NAME" "$NIGHLTY_DIR_RSYNC_URL/"
```

Comment:
The idea was actually to remove the second parameter, like in `upload_build_artifacts` and `upload_iso`. Fixing.